invenio_app_rdm/theme/assets/js/react_invenio_forms/core/FieldLabel.js

```diff
@@ -0,0 +1,25 @@
+// This file is part of React-Invenio-Forms
+// Copyright (C) 2020 CERN.
+// Copyright (C) 2020 Northwestern University.
+//
+// React-Invenio-Forms is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { Icon } from "semantic-ui-react";
+
+export class FieldLabel extends Component {
+  render() {
+    const {htmlFor, icon, label} = this.props;
+    return (
+      <label htmlFor={htmlFor}>{icon ? <Icon name={icon} /> : null}{label}</label>
+    )
+  }
+}
+
+FieldLabel.propTypes = {
+  htmlFor: PropTypes.string,
+  label: PropTypes.string,
```

Comment:
Just mark the props that are required here, e.g `label`, `htmlFor`. Also a nice trick for optional props that you want to give a default value is this https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values

Reply:
I thought `propTypes` was a nice way to define the interface for the component, but it should only be used for required props? So if a prop is not required, it should not be placed in the `propTypes`but rather only be placed in the `defaultProps`? Did I get this right?

What if there is no sensible default?: for icon there isn't a default - I guess `null` or `undefined` could be in the default, but kind of implied...